src/AWS/Orleans.Reminders.DynamoDB/Reminders/DynamoDbReminderServiceOptions.cs

```diff
@@ -14,29 +14,7 @@ public class DynamoDBReminderTableOptions
         /// <summary>
         /// Gets or sets the connection string.
         /// </summary>
+        [RedactConnectionString]
```

Comment:
Are we confident RedactConnectionString is safe with DynamoDb connection string?

Reply:
It is explicitly handled in `ConfigUtilities.RedactConnectionStringInfo`